matyas.brtan.s@P1-16 MINGW64 ~
$ cd c:

matyas.brtan.s@P1-16 MINGW64 /c
$ cd users

matyas.brtan.s@P1-16 MINGW64 /c/users
$ cd matyas.brtan.s

matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s
$ cd Desktop

matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop
$ cd 2024_wt_prj_brtan

matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_pmain)
$ cd prj

matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ py -3 -m venv venv

matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ source ./venv/Scripts/activate
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ ./manage.py runserver
Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p1, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p2, in <module>
    main()
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p3, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and an your PYTHONPATH environment variable? Did you forget to activate a vironment?
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ cd manage.py
bash: cd: manage.py: Not a directory
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ ./manage.py runserver
Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p1, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p2, in <module>
    main()
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p3, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and an your PYTHONPATH environment variable? Did you forget to activate a vironment?
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ cd prj
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj/prj (main)
$ ./manage.py runserver
bash: ./manage.py: No such file or directory
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj/prj (main)
$ cd ..
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ ./manage.py runserver
Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p1, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p2, in <module>
    main()
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p3, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and an your PYTHONPATH environment variable? Did you forget to activate a vironment?
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ import django
bash: import: command not found
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ pip install -r requirements
ERROR: Could not open requirements file: [Errno 2] No such file or direquirements'

[notice] A new release of pip is available: 24.0 -> 25.0.1
[notice] To update, run: python.exe -m pip install --upgrade pip
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ ./manage.py runserver
Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p
    main()
  File "C:\users\matyas.brtan.s\Desktop\2024_wt_prj_brtan\prj\manage.p
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and a
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_p
$ ^C
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_p
$ pip list
Package Version
------- -------
pip     24.0

[notice] A new release of pip is available: 24.0 -> 25.0.1
[notice] To update, run: python.exe -m pip install --upgrade pip
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_p
$ pip install django
Collecting django
  Downloading Django-5.1.7-py3-none-any.whl.metadata (4.1 kB)
Collecting asgiref<4,>=3.8.1 (from django)
  Using cached asgiref-3.8.1-py3-none-any.whl.metadata (9.3 kB)
Collecting sqlparse>=0.3.1 (from django)
  Using cached sqlparse-0.5.3-py3-none-any.whl.metadata (3.9 kB)
Collecting tzdata (from django)
  Using cached tzdata-2025.1-py2.py3-none-any.whl.metadata (1.4 kB)
Downloading Django-5.1.7-py3-none-any.whl (8.3 MB)
   ---------------------------------------- 8.3/8.3 MB 10.0 MB/s eta 0
Using cached asgiref-3.8.1-py3-none-any.whl (23 kB)
Using cached sqlparse-0.5.3-py3-none-any.whl (44 kB)
Using cached tzdata-2025.1-py2.py3-none-any.whl (346 kB)
Installing collected packages: tzdata, sqlparse, asgiref, django
Successfully installed asgiref-3.8.1 django-5.1.7 sqlparse-0.5.3 tzdat

[notice] A new release of pip is available: 24.0 -> 25.0.1
[notice] To update, run: python.exe -m pip install --upgrade pip
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ source ./venv/Scripts/activate
(venv)
matyas.brtan.s@P1-16 MINGW64 /c/users/matyas.brtan.s/Desktop/2024_wt_prj (main)
$ ./manage.py runserver
